### PR TITLE
Add console route slo document [WIP]

### PR DESCRIPTION
**Pull request by sulmer-redhat**
/hold

### CODE REVIEW

**Comment by sulmer-redhat** [comment]
I am currently testing the SLI with the following steps:
  a. New staging cluster
  b. Access the console of the new staging cluster 3 times or more a day
  c. Do this for a week and make sure that the availability is over 99.5%.

Is a window of 1 week enough to test on a staging cluster to verify SLO wasn't breached?

**Comment by cblecker** [comment]
Is this the right repo for this?

**Comment by sulmer-redhat** [comment]
@RiRa12621 @NautiluX You asked me to put it here in "managed cluster config", right?

**Comment by sulmer-redhat** [comment]
> Is this the right repo for this?

**Comment by sulmer-redhat** [comment]
reopen

**Comment by RiRa12621** [comment]
additional proposals on where it should go @cblecker ?
We were not sure to be honest
So ideally in the future each SLO document should live close to it's component but it needs to be accepted by the component owner then and so on so for now while it's OSD specific, we thought it should live here

File: docs/SLO-Documents/OSD_Console_Route_SLO.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+##### **Error Budget**
+
+Each objective has a separate error budget, defined as 100% minus (–)
+the goal for that objective. For example, if there have been 1,000,000
+requests to the API server in the previous four weeks, the API
+availability error budget is 3% (100% – 97%) of 1,000,000: 30,000
+errors.
+
+We will enact the error budget policy (see [Document](https://landing.google.com/sre/workbook/chapters/error-budget-policy/))
```
Comment (RiRa12621) [inline docs/SLO-Documents/OSD_Console_Route_SLO.md:50]: Are we **really** enacting that policy or any at all?

File: docs/SLO-Documents/OSD_Console_Route_SLO.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+##### **Service Overview**
+
+The console route exists within the Openshift Ingress Router, and is enabling the access of end users and OSD SRE team to the web UI in order to manage OSD workload.
+The SLO uses a four-week rolling window.
+
+##### **SLIs and SLOs**
+
+| Category     | SLI                                                                                                                                                                     | SLO                                                      | Prometheus Query |
+|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|------------------|
+| Availability | The proportion of successful requests, as measured from the <SERVICE\|ROUTER\|ELB> metrics.                                                                             | 99.5% Success (3.36 Hours of error budget)                                              |                  |
```
Comment (RiRa12621) [inline docs/SLO-Documents/OSD_Console_Route_SLO.md:25]:
```suggestion
| Availability | The proportion of successful requests, as measured from the Router metrics.                                                                             | 99.5% Success (3.36 Hours of error budget) 
```

Comment (RiRa12621) [inline docs/SLO-Documents/OSD_Console_Route_SLO.md:25]: we measure on the router

File: docs/SLO-Documents/OSD_Console_Route_SLO.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+This document describes the SLOs for the ReplaceMeService.
+
+| **Status**    | **Work In Progress**                        |
+| ------------- | ------------------------------------------- |
+| Author        | Shay Ulmer                                  |
+| Date          | 29/07/2020                                  |
+| Reviewers     | Manuel Dewald, Rick Rackow, Pat Cremin      |
+| Approvers     | \<Service Owner\>                           |
```
Comment (RiRa12621) [inline docs/SLO-Documents/OSD_Console_Route_SLO.md:12]: needs approval
while it's sre-exclusive, that's @patrickstrick + team leads

**Comment by RiRa12621** [comment]
/cc @pat-cremin @NautiluX 
/assign @patrickstrick 
we need the acceptance from service owner before going ahead

**Review by NautiluX — COMMENTED** [review]
Let's close this PR in favor of the new home for SLO docs in Google Drive.

**Pull request closed without merging**